packages/@aws-cdk-testing/framework-integ/test/aws-eks/test/integ.eks-helm-asset.js.snapshot/aws-cdk-eks-helm-test.assets.json

```diff
@@ -118,15 +118,15 @@
         }
       }
     },
-    "87e05e7e56d00f8bd9133b97480ff01b622c9f7f500f8544cef311a412454395": {
+    "0ab01c1684677dce72948d48cdd47d31aef3137c741b066737be81fe5944e6fe": {
```

Comment:
This asset hash update means that there should also be changes to the asset files. At the very least, they should be renamed.